sport/app/football/feed/Competitions.scala

```diff
@@ -402,4 +407,16 @@ class CompetitionsService(val footballClient: FootballClient, competitionDefinit
     })
     result.map(_.flatten).flatMap(Future.sequence(_))
   }
+
+  def maybeRefreshLiveMatches(
+      clock: Clock,
+  )(implicit executionContext: ExecutionContext): Future[immutable.Iterable[Competition]] = {
+    // matches is the list of all matches from all competitions
+    if (isMatchLiveOrAboutToStart(matches, clock)) {
```

Comment:
maybe we could rename the val to allMatches

Reply:
That's a great idea. If that's OK I'll do this in a separate PR because changing this val would cause many file changes as it's being referenced in several files and it may make it harder to understand the purpose of this PR?